The cross-build is broken

```
I0815 09:30:09.341] +++ [0815 09:28:58] windows/amd64: go build started
I0815 09:30:09.341] # k8s.io/kubernetes/pkg/kubelet/dockershim
I0815 09:30:09.341] pkg/kubelet/dockershim/docker_container.go:153: not enough arguments in call to ds.getSecurityOpts
I0815 09:30:09.341] 	have (string, rune)
I0815 09:30:09.342] 	want (string, *"k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime".PodSandboxConfig, rune)
I0815 09:30:09.342] pkg/kubelet/dockershim/docker_sandbox.go:591: not enough arguments in call to ds.getSecurityOpts
I0815 09:30:09.342] 	have (string, rune)
I0815 09:30:09.342] 	want (string, *"k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime".PodSandboxConfig, rune)
I0815 09:30:09.342] !!! [0815 09:30:09] Call tree:
I0815 09:30:09.343] !!! [0815 09:30:09]  1: /go/src/k8s.io/kubernetes/hack/lib/golang.sh:697 kube::golang::build_binaries_for_platform(...)
I0815 09:30:09.343] !!! [0815 09:30:09]  2: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
I0815 09:30:09.348] Makefile:92: recipe for target 'all' failed
I0815 09:30:09.351] Makefile:419: recipe for target 'cross' failed
W0815 09:30:09.452] !!! [0815 09:30:09] Call tree:
W0815 09:30:09.452] !!! [0815 09:30:09]  1: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
W0815 09:30:09.452] make[1]: *** [all] Error 1
W0815 09:30:09.453] make: *** [cross] Error 1
W0815 09:30:09.743] !!! [0815 09:30:09] Call tree:
W0815 09:30:09.745] !!! [0815 09:30:09]  1: build/../build/common.sh:521 kube::build::run_build_command_ex(...)
W0815 09:30:09.747] !!! [0815 09:30:09]  2: build/release.sh:36 kube::build::run_build_command(...)
W0815 09:30:09.760] make: *** [release] Error 1
W0815 09:30:09.761] Traceback (most recent call last):
W0815 09:30:09.761]   File "/var/lib/jenkins/workspace/pull-kubernetes-cross/./test-infra/jenkins/../scenarios/execute.py", line 50, in <module>
W0815 09:30:09.784]     main(ARGS.env, ARGS.cmd + ARGS.args)
W0815 09:30:09.784]   File "/var/lib/jenkins/workspace/pull-kubernetes-cross/./test-infra/jenkins/../scenarios/execute.py", line 41, in main
W0815 09:30:09.785]     check(*cmd)
W0815 09:30:09.785]   File "/var/lib/jenkins/workspace/pull-kubernetes-cross/./test-infra/jenkins/../scenarios/execute.py", line 30, in check
W0815 09:30:09.785]     subprocess.check_call(cmd)
W0815 09:30:09.785]   File "/usr/lib/python2.7/subprocess.py", line 186, in check_call
W0815 09:30:09.802]     raise CalledProcessError(retcode, cmd)
W0815 09:30:09.802] subprocess.CalledProcessError: Command '('make', 'release')' returned non-zero exit status 2
I0815 09:30:09.809] Makefile:382: recipe for target 'release' failed
E0815 09:30:09.809] Command failed
```

seems to belong to sig node @kubernetes/sig-node-bugs PTAL and fix ASAP

--- a/pkg/kubelet/dockershim/helpers_windows.go
+++ b/pkg/kubelet/dockershim/helpers_windows.go
@@ -26,7 +26,6 @@ import (
 	dockercontainer "github.com/docker/docker/api/types/container"
 	dockerfilters "github.com/docker/docker/api/types/filters"
 	"github.com/golang/glog"
-	"k8s.io/api/core/v1"
 	runtimeapi "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
 )
 
@@ -34,19 +33,10 @@ func DefaultMemorySwap() int64 {
 	return 0
 }
 
-func (ds *dockerService) getSecurityOpts(containerName string, sandboxConfig *runtimeapi.PodSandboxConfig, separator rune) ([]string, error) {
-	hasSeccompSetting := false
-	annotations := sandboxConfig.GetAnnotations()
-	if _, ok := annotations[v1.SeccompContainerAnnotationKeyPrefix+containerName]; !ok {
-		_, hasSeccompSetting = annotations[v1.SeccompPodAnnotationKey]
-	} else {
-		hasSeccompSetting = true
+func (ds *dockerService) getSecurityOpts(seccompProfile string, separator rune) ([]string, error) {
+	if seccompProfile != "" {
+		glog.Warningf("seccomp annotations are not supported on windows")
 	}
-
-	if hasSeccompSetting {
-		glog.Warningf("seccomp annotations found, but it is not supported on windows")
-	}
-
 	return nil, nil
 }
 

